lib/node_modules/@stdlib/assert/has-bigint-support/lib/main.js

```diff
@@ -20,12 +20,7 @@
 
 // MODULES //
 
-var getGlobal = require( '@stdlib/utils/global' );
```

Comment:
This is most definitely not the change we are looking for. You need to revert these changes. Instead, for L45 and L47, you simply need to add `stdlib/no-builtin-big-int` to the list of lint rules which are disabled. It is intentional for us to use the `BigInt` global in this file. Importing a module defeats that purpose.